History.md

```diff
@@ -1,4 +1,9 @@
 
+0.2.2 / 2016-07-22
```

Comment:
Please add this as "Pending release", that way I can change the date and version when I cut the actual release